packages/core/src/components/forms/numericStepper.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import * as Classes from "../../common/classes";
+import * as Keys from "../../common/keys";
+import { removeNonHTMLProps } from "../../common/props";
+import { HTMLInputProps, IControlledProps, IIntentProps, IProps } from "../../common/props";
+
+import { Button } from "../button/buttons";
+import { InputGroup } from "./inputGroup";
+
+export type ButtonPosition = "none" | "left" | "right" | "split";
+
+export interface INumericStepperProps extends IControlledProps, IIntentProps, IProps {
+
+    /**
+     * The button configuration with respect to the input field.
+     * @default right
+     */
+    buttonPosition?: ButtonPosition;
+
+    /**
+     * Whether the input is non-interactive.
+     * @default false
+     */
+    disabled?: boolean;
+
+    /** Ref handler that receives HTML `<input>` element backing this component. */
+    inputRef?: (ref: HTMLInputElement) => any;
+
+    /** Name of icon (the part after `pt-icon-`) to render on left side of input. */
+    leftIconName?: string;
+
+    /** Placeholder text in the absence of any value. */
+    placeholder?: string;
+
+    /**
+     * Increment between successive values when "Shift" is held.
+     * @default 10
+     */
+    majorStepSize?: number;
+
+    /** Maximum value of the input. */
+    max?: number;
+
+    /** Minimum value of the input. */
+    min?: number;
+
+    /**
+     * Increment between successive values when "Alt" is held.
```

Comment:
what other props tables? i think <kbd>alt</kbd> is the accepted name for this key (mainly cuz there's no <kbd>option</kbd> on windows.